packages/cli/test/unit/commands/list.test.ts

```diff
@@ -35,11 +35,13 @@ describe('list', () => {
 
       await list(client);
 
-      const output = await readOutputStream(client);
+      const output = await readOutputStream(client, 4);
+      console.log(output);
```

Comment:
Is this console.log() supposed to be here?

Reply:
Nope, thanks for catching that